app.rb

```diff
@@ -90,5 +91,5 @@ def verify_signature(payload_body)
 end
 
 def redmine_issue_repos
-  ENV['REDMINE_ISSUE_REQUIRED_REPOS'].to_s.split.map { |r| Regexp.new("\\A#{r}\\z") }
+  YAML.load_file('config/redmine_issue_required_repos.yaml')
```

Comment:
It was just as a means of keeping the list shorter.  I don't think it could have been added for smart_proxy_remote_exec.*, but it's likely that we had something else in the list that has since been removed, and that I've since forgotten.  Could it have been `katello/.*`?  I'm not sure.

.include? would pose a problem with "theforeman/foreman" since many repos have that prefix.